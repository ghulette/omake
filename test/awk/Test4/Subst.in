hello.world.hi again.once more
