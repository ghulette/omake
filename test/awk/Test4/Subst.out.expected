Hello.World.hi Again.once more
